learn/getting_started/installation.md

```diff
@@ -35,6 +35,7 @@ meilisearch
 :::
 
 ::: tab Docker
+If you are new to Docker, you can learn more about setting it up on the [official documentation](https://docs.docker.com/get-docker/).
 Using **Docker** you can choose to run [any available tags](https://hub.docker.com/r/getmeili/meilisearch/tags).
```

Comment:
These two sentences feel a bit disconnected, as if they were forced together. Do you think we can improve it?

Reply:
I guess a related issue is the order here. We first send people to learn about docker before saying why they should do it.

Reply:
updated the order

Reply:
Remember to resolved the conversation =)